packages/design-system/src/components/OcModal/OcModal.vue

```diff
@@ -288,7 +288,7 @@ const initialFocusRef = computed<FocusTargetOrFalse>(() => {
     return focusTrapInitial as FocusTargetOrFalse
   }
 
-  return () => unref(ocModalInput)?.$el || unref(ocModal)
+  return () => false
```

Comment:
The focus should still be sent somewhere in the dialog. With this change, the focus is sent to body. That results in "Skip to main" button being the next element which will be focused.